app/src/main/java/com/android/brewr/ui/userProfile/UserMainProfile.kt

```diff
@@ -47,14 +47,14 @@ import com.android.brewr.model.user.UserViewModel
 import com.android.brewr.ui.navigation.NavigationActions
 import com.android.brewr.ui.navigation.Route
 import com.android.brewr.ui.navigation.Screen
+import com.android.brewr.ui.theme.CoffeeBrown
+import com.android.brewr.ui.theme.LightBrown
 import com.google.firebase.auth.ktx.auth
 import com.google.firebase.ktx.Firebase
 
 @OptIn(ExperimentalMaterial3Api::class)
 @Composable
 fun UserMainProfileScreen(userViewModel: UserViewModel, navigationActions: NavigationActions) {
-  val NOT_YET_IMPLEMENTED = "Feature not yet developed"
```

Comment:
Great job. Since our last Milestone arrives soon, it's better to remove all the "To be implemented" features rather than let them as "Not implemented yet". This was one of my remark during our standup meetings. And I'm glad that you have applied my feedback !